File: Source/C#/RiceDoctor/RiceDoctor.OntologyManager/Models/DataType.cs
using Newtonsoft.Json;
using Newtonsoft.Json.Converters;

namespace RiceDoctor.OntologyManager
{
    [JsonConverter(typeof(StringEnumConverter))]
    public enum DataType
    {
        Unknown,
        Enumerated,
        Int,
        String,
        Boolean
    }
}

File: Source/Java/src/main/java/vn/edu/uit/ontologymanager/model/DataType.java
package vn.edu.uit.ontologymanager.model;

import com.google.gson.annotations.SerializedName;

public enum DataType {

    @SerializedName("Unknown")
    UNKNOWN,

    @SerializedName("Enumerated")
    ENUMERATED,

    @SerializedName("Int")
    INT,

    @SerializedName("String")
    STRING,

    @SerializedName("Boolean")
    BOOLEAN
}
